AUTHORS

```diff
@@ -512,3 +513,17 @@ Nathan Musoke <nathan.musoke@gmail.com>
 Abhishek Garg <abhishekgarg119@gmail.com>
 Dana Jacobsen <dana@acm.org>
 Vasiliy Dommes <vasdommes@gmail.com>
+Phillip Berndt <phillip.berndt@googlemail.com>
+Haimo Zhang <zh.hammer.dev@gmail.com>
+Subham Tibra <shubh.tibra@gmail.com>
```

Comment:
I thought there was an "h" in his name. Good to know I wasn't just misremembering that. @shubhamtibra which spelling do you prefer?